x/ibc-rate-limit/.beaker/state.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
should this file be in the gitignore?

Reply:
Not sure. I just used the defaults from beaker which have `state.local.json` in the gitignore, but not state.json. @iboss-ptk is there any reason to commit `.beaker/state.json`?

Reply:
intentionally keep tracks of public chain so for example, if another contributor wants to interact with / migrate deployed contract, they have all the refs in place.

Reply:
This contract should be getting deployed in a future Osmosis upgrade

Reply:
Right, so IIUC this will only have information if the contract is deployed via beaker and this file is committed afterwards. On the one hand, we would have information such as the code_id/contract_addr there; on the other, no one will probably use this directly since all interactions with the contract happen via governance (except maybe getting the current state, but I'm guessing people easily can craft that query manually). 

If that's right, I think it makes sense to delete this.

Reply:
removed and added to the gitignore.